### PR TITLE
Adds start.py

### PR DESCRIPTION
**Pull request by u9g**
Syntax example:
```
yarn3> python ./start.py yarn --help
Syntax:
    python ./start.py action mcVersion metaUrl
Example:
    python ./start.py yarn 1.8.9
    python ./start.py yarn 1.7.10 https://meta.legacyfabric.net
    python ./start.py publishlocal 1.8.9
```

Makes it much simpler to publish mappings or open yarn.

### CODE REVIEW

File: start.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+HELP_STR = """Syntax:
+    python ./start.py action mcVersion metaUrl
+Example:
+    python ./start.py yarn 1.8.9
+    python ./start.py yarn 1.7.10 https://meta.legacyfabric.net
+    python ./start.py publishlocal 1.8.9"""
+
+def main():
+    args = sys.argv
+    if len(set(["--help", "-h"]) & set(args))>0:
```
Comment (ENDERZOMBI102) [inline start.py:13]: Why didn't you use the [argparse](https://docs.python.org/3/library/argparse.html#module-argparse) module?

File: start.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        print("Version wasn't specified, input should look like:")
+        print(HELP_STR)
+        return
+    action = args[1]
+    version = args[2]
+    if len(args) == 4:
+        os.environ['LEGACY_FABRIC_META_URL'] = args[3]
+    if action == 'yarn':
+        os.environ['MC_VERSION'] = version
+        os.system('gradlew.bat yarn')
```
Comment (ENDERZOMBI102) [inline start.py:30]: Inconsistency, why does this use the `.bat` version while the other doesn't?
Both should be the same, and based off the user's running OS as Windows requires the `.bat` and unix needs the one without

File: start.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    elif action == 'publishlocal':
+        os.system('python ./prepare_yarn.py ' + version)
+        os.system('gradlew publishToMavenLocal --stacktrace')
+
+if __name__ == '__main__':
+    main()
```
Comment (ENDERZOMBI102) [inline start.py:36]: There should be a newline at the end of the file

File: start.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+
```
Comment (ENDERZOMBI102) [inline start.py:3]: there should be two newlines between blocks of code, here's there is only one

File: build.gradle [hunk 5]
```diff
@@ -165,7 +166,7 @@ task downloadWantedVersionManifest(dependsOn: downloadVersionsManifest) {
 		if (manifestVersion.isPresent() || versionFile.exists()) {
 
 			if (manifestVersion.isPresent()) {
-				FileUtils.copyURLToFile(new URL("https://meta.legacyfabric.net/v2/manifest/${manifestVersion.get().version}"), versionFile)
+				FileUtils.copyURLToFile(new URL(metaURL + "/v2/manifest/${manifestVersion.get().version}"), versionFile)
```
Comment (ENDERZOMBI102) [inline build.gradle:169]: use templated string, as already used.
```groovy
"$metaURL/v2/manifest/${manifestVersion.get().version}"
```

**Review by ENDERZOMBI102 — CHANGES_REQUESTED** [review]
please fix

**Comment by BleachDrinker420** [comment]
My suggestions:
- Rename to yarn.py and delete the old bootstrap files
- Show the list of default versions and give the user a choice like the old bootstrap if theres no arguments (plus a choice whether to edit, build or publish)

File: start.py (hunk 1 above)
Comment (u9g) [reply start.py:14]: I didn't feel that it's necessary, and still don't

File: start.py (hunk 2 above)
Comment (u9g) [reply start.py]: fixed in ead1491

File: start.py (hunk 3 above)
Comment (u9g) [reply start.py]: fixed in 8ef87e6

File: build.gradle (hunk 5 above)
Comment (u9g) [reply build.gradle]: done in 1d8c0e0

**Comment by u9g** [comment]
> My suggestions:
> 
> * Rename to yarn.py and delete the old bootstrap files
> * Show the list of default versions and give the user a choice like the old bootstrap if theres no arguments (plus a choice whether to edit, build or publish)

1. Done
2. I did edit/publish, what do you mean by build though? just run ./gradlew build instead of publishToMavenLocal?

**Comment by BleachDrinker420** [comment]
Yea build just builds it but guess it isn't really needed

**Comment by BleachDrinker420** [comment]
Also i meant that when the user just clicks the file they can input which version they want like in the old bootstrap, i edited the script a bit to make it work

https://pastebin.com/vr4p2y71

**Comment by u9g** [comment]
> Also i meant that when the user just clicks the file they can input which version they want like in the old bootstrap, i edited the script a bit to make it work
> 
> https://pastebin.com/vr4p2y71

sounds good

**Review by BleachDrinker420 — APPROVED** [review]

**Review by arthurbambou — APPROVED** [review]

**Comment by arthurbambou** [comment]
Superseeded by #196

**Pull request closed without merging**